.github/workflows/CD.yml

```diff
@@ -0,0 +1,116 @@
+name: CD
+
+on:
+  push:
+    tags:        
+      - '**' 
+
+jobs:
+  validate-tag:
+    name: Validate tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+          fetch-depth: 0
+      
+      - name: Check no jumps in tags
+        run: |
+          # If a previous tag is X.Y, valid tags are only X.(Y+1) or (X+1).0
+          if [[ $(git tag -l | wc -l) != 1 ]]; then # If there is only one tag, no need to check
+            prev=($(git tag -l --sort=-v:refname | head -n 2 | tail -n 1 | tr "." " "))
+            val1="${prev[0]}.$((${prev[1]}+1))" # X.(Y+1)
+            val2="$((${prev[0]}+1)).0" # (X+1).0
+            if [[ "${{github.ref_name}}" != "$val1" ]] && \
+            [[ "${{github.ref_name}}" != "$val2" ]]; then
+                echo "Invalid tag \"${{github.ref_name}}\" when previous tag is 
+                \"$(sed 's/ /./' <<< ${prev[@]})\"."
+                exit 1
+            fi
+          fi
+  
+  get-package-name:
+    name: Get package name
+    runs-on: ubuntu-latest
+    needs: validate-tag
+    outputs:
+      package-name: ${{ steps.get-package-name.outputs.package-name }}
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+          fetch-depth: 0
+      
+      - name: Get name
+        id: get-package-name
+        run: |
+          echo "package-name=$(yq -oy '.project.name' pyproject.toml)" >> $GITHUB_OUTPUT
```

Comment:
What is the `-y` flag? I can't find it in `yq --help`. And the `-o` isn't nessecary if you don't give a output type (defaults to `auto`)

Reply:
For some reason I remember getting an error if I tried to parse the `pyproject.toml` file with `yq`, and reading I had to use the `-oy` options to make it work. 
But it seems to work also without any option. 
Not sure why I added in the first place, but I removed it in [210e00add46f0c74d626e77b540e276a4d8b6609](https://github.com/ACCESS-NRI/replace_landsurface/pull/45/commits/210e00add46f0c74d626e77b540e276a4d8b6609)